src/google/protobuf/compiler/cpp/cpp_bootstrap_unittest.cc

```diff
@@ -60,32 +60,41 @@
 #include <google/protobuf/testing/googletest.h>
 #include <gtest/gtest.h>
 
-namespace google {
-namespace protobuf {
-namespace compiler {
-namespace cpp {
-
-namespace {
-
-class MockErrorCollector : public MultiFileErrorCollector {
- public:
+namespace google
```

Comment:
Can you undo the formatting changes? The existing formatting matches our coding style so we would prefer to keep the existing formatting.

Reply:
Oh, I didn't notice this. I just run the `format ` in my local IDE :(
Fixed